core/block_validator.go

```diff
@@ -127,3 +133,17 @@ func CalcGasLimit(parentGasLimit, desiredLimit uint64) uint64 {
 	}
 	return limit
 }
+
+// VerifyCalldataCap calculates the total calldata size of the block's transactions and
+// verifies that it is below the cap introduced by EIP-XXXX
```

Comment:
Why is this code using EIP4448 instead of EIPXXXX for variable names and comments?